cmd/juju/machine/upgradeseries.go

```diff
@@ -71,7 +77,7 @@ run beforehand.
 
 Examples:
 	juju upgrade-series prepare <machine> <series>
```

Comment:
at some point the examples should match the new  CLI help text formatting examples.

Reply:
+1 please reference https://docs.google.com/document/d/1ySjCNqd0x6veLfcBetxLI9NH7qfw3xayLWJNqXMvyW8/edit